JS/Lesson15/sinan/script.js

```diff
@@ -116,14 +129,49 @@ function checkIfStringIsUpperCase(stringToCheck) {
 	}
 }
 
-checkIfStringIsUpperCase('Write a program that takes in');
-checkIfStringIsUpperCase('write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.1:','Write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.2:','write a program that takes in');
 
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function checkNumberPositive(checkNumber) {
```

Comment:
I'd rename like this: 
- 'checkNumberPositive' -> 'checkNumberSign` 
- 'checkNumber' -> 'number'